apps/daimo-mobile/src/logic/log.ts

```diff
@@ -41,6 +41,30 @@ export class Log {
     }
   }
 
+  static async retryBackoff<T>(
+    type: string,
+    fn: () => Promise<T>,
+    retries: number,
+    matchError?: (e: string) => boolean
+  ) {
+    for (let i = 0; i < retries; i++) {
+      try {
+        return await this.promise(type, fn());
+      } catch (e: any) {
+        if (matchError && matchError(getErrMessage(e))) {
+          console.log(`[LOG] ${type} trial ${i} error ${getErrMessage(e)}`);
+          await new Promise((r) => setTimeout(r, 200 * 2 ** i));
+        } else {
+          console.log(
+            `[LOG] ${type} skipping retry ${i} error ${getErrMessage(e)}`
```

Comment:
nit to shorten `type} trial ${i}, quitting ...` (we don't skip, but quit trying altogether)

Reply:
done